terraform/modules/nitro/variables.tf

```
@@ -0,0 +1,9 @@
+variable "instance_name" {
+  description = "The name of the instance"
+  type        = string
+}
+
+variable "key_pair_name" {
+  description = "The name of the key pair"
```

Might be worth adding a few words to communicate that it is specific to the name of the pair in EC2 and must exist already in the region you are deploying
```suggestion
variable "key_pair_name" {
  description = "The name of the key pair in AWS. This key pair must exist in the region you are deploying to."
```